sonar-db/src/main/java/org/sonar/db/version/Select.java

```diff
@@ -25,7 +25,6 @@
 import java.sql.Timestamp;
 import java.util.Date;
 import java.util.List;
-
 import javax.annotation.CheckForNull;
 
 public interface Select extends SqlStatement<Select> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1214)